### ISSUE TITLE
predict代码问题

### ISSUE THREAD

**Issue by 15812463834** [Wzysaber/ST_Unet_pytorch_Semantic-segmentation#2]
你好，请问一下这里预测代码为啥是用UNET来的，
 dataloader = DataLoader(dataset=dataset, batch_size=1, shuffle=False, num_workers=0)
 model = Unet(num_classes=6)
 state_dict = torch.load(args.model_path)，我现在已经训练好了，但是这部分预测miou和f1score都很低，并不像训练时那么高，是什么原因

**Comment by lovedyh1314** [comment]
> 没有解决.......换课题了
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2025年1月7日(星期二) 晚上6:10 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [Wzysaber/ST_Unet_pytorch_Semantic-segmentation] predict代码问题 (Issue #2) 你好，请问一下这里预测代码为啥是用UNET来的， dataloader = DataLoader(dataset=dataset, batch_size=1, shuffle=False, num_workers=0) model = Unet(num_classes=6) state_dict = torch.load(args.model_path)，我现在已经训练好了，但是这部分预测miou和f1score都很低，并不像训练时那么高，是什么原因 我的predict预测miou和f1score都超级低，学长您解决了吗 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

好的，谢谢您，我也暂时没有解决
